DESCRIPTION

```diff
@@ -1,9 +1,26 @@
 Package: str.rdev
 Title: What the Package Does (One Line, Title Case)
 Version: 0.0.0.9000
-Authors@R: NULL
+Authors@R: person("Rex", "Parsons", role = c("aut", "cre"),
+                  email = "rex.parsons94@gmail.com",
+                  comment = c(ORCID = "0000-0002-6053-8174"))
 Description: What the package does (one paragraph).
 License: GPL (>= 3)
 Encoding: UTF-8
 Roxygen: list(markdown = TRUE)
 RoxygenNote: 7.2.3
+Suggests: 
+    ggplot2,
+    scales,
+    shiny,
+    shinyWidgets,
+    testthat (>= 3.0.0)
+Config/testthat/edition: 3
+Imports: 
+    dplyr,
+    purrr,
+    readxl,
+    stringr,
```

Comment:
Thanks. I didn't focus too much on reducing dependencies but I could have swapped the `purrr::map()`'s for some `lapply()`'s and the `stringr::str_remove()` for a `gsub()` to avoid those two pretty easily.